modules/cnf-numa-resource-scheduling-strategies.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_module-type: CONCEPT
+[id="cnf-numa-resource-scheduling-strategies_{context}"]
+= NUMA Resource scheduling strategies 
+
+When scheduling high-performance workloads, the secondary scheduler can employ different strategies to determine which NUMA node within a chosen worker node will handle the workload. The supported strategies in {product-title} include `LeastAllocated`, `MostAllocated`, and `BalancedAllocation`. Understanding these strategies helps optimize workload placement for performance and resource utilization.
+
+When a high-performance workload is scheduled in a NUMA-aware cluster, the following steps occur: 
+
+. The scheduler first selects a suitable worker node based on cluster-wide criteria for example taints, labels, or resource availability.
+
+. After a worker node is selected, the scheduler evaluates its NUMA nodes and applies a scoring strategy to decide which NUMA node will handle the workload.
+
+. Once a workload is scheduled, the selected NUMA node’s resources are updated to reflect the allocation.
+
+The default strategy applied is the `LeastAllocated` strategy. This assigns workloads to the NUMA node with the most available resources that is the least utilized NUMA node. The goal of this strategy is to spread workloads across NUMA nodes to reduce contention and avoid hotspots.
+
+.Example
```

Comment:
Maybe for each of these three `.Example` titles describing the strategies in more detail, use a more descriptive title for each, e.g.:
```suggestion
.`LeastAllocated` example
```
or better yet, switch to an actual subsection heading (but maybe use `discrete` if you want to avoid showing up in the TOC):
```suggestion
[discrete]
[id="cnf-leastallocated-example_{context}"]
== LeastAllocated example
```
^ note lack of backticks if you do switch them all to `==` headings.